src/txmempool.cpp

```diff
@@ -875,6 +870,22 @@ void CTxMemPool::ClearPrioritisation(const uint256 hash)
     mapDeltas.erase(hash);
 }
 
+const CTransaction* CTxMemPool::GetConflictTx(const COutPoint& prevout) const
+{
+    const auto it = mapNextTx.find(prevout);
+    return it == mapNextTx.end() ? nullptr : it->second;
+}
+
+CTxMemPool::setEntries CTxMemPool::GetIterSet(const std::set<uint256>& hashes) const
+{
+    CTxMemPool::setEntries ret;
```

Comment:
as the result is only ever used to iterate over, does this need to return a set? I guess a std::list would be a better match

Reply:
OK I guess I'm wrong, there might be duplicate parents